openssl/src/cms.rs

```diff
@@ -9,11 +9,39 @@ use ffi;
 use foreign_types::{ForeignType, ForeignTypeRef};
 use std::ptr;
 
-use {cvt, cvt_p};
 use bio::{MemBio, MemBioSlice};
 use error::ErrorStack;
 use pkey::{HasPrivate, PKeyRef};
+use stack::Stack;
 use x509::X509;
+use {cvt, cvt_p};
+
+bitflags! {
+    pub struct CMSOptions : u32 {
+        const CMS_TEXT = 0x1;
```

Comment:
These literals should be defined in the sys crate so we can confirm their correctness.

Let's also leave out the `CMS_` prefix for the bitflags constant names.